package.json

```diff
@@ -27,7 +27,7 @@
         "fetch": "npm-run-all --parallel fetch:*",
         "start": "npm-run-all build:sass --parallel watch:*",
         "build": "npm-run-all install:playground build:sass build:eleventy build:webpack images",
-        "lint": "eslint --ext=.js,.jsx .",
+        "lint": "eslint --ext=.js,.jsx . && markdownlint \"**/*.md\" -i node_modules",
```

Comment:
I think we can introduce a separate command for linting markdown? something like `lint:md` and then use use `npm-run-all` in parallel to lint js and md files?